Tests/Tests/CommonUtilitySpec.swift

```diff
@@ -104,6 +104,38 @@ class CommonUtilitySpec: QuickSpec {
                 }
             }
 
+            context("CommonUtility.convertTriggerObjectToCustomEvent") {
+
+                it("will return Custom Event") {
+                    let attribute = TriggerAttribute(name: "att", value: "val",
+                                                     type: .string, operatorType: .isNotBlank)
+                    let trigger = Trigger(type: .event, eventType: .appStart, eventName: "app Start", attributes: [attribute])
+                    let result = CommonUtility.convertTriggerObjectToCustomEvent(trigger)
+                    expect(result).toNot(beNil())
```

Comment:
`convertTriggerObjectToCustomEvent()` doesn't return an optional type so it doesn't make sense to check if `result` is not nil. Instead, it would be better to test if returned `CustomEvent` object has the same name and attributes as the Trigger object